swift_browser_ui/ui/server.py

```diff
@@ -2,6 +2,8 @@
 
 # Generic imports
 import logging
+import os.path
+import pickle  # nosec
```

Comment:
importing packages with nosec not sure given that https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html it is imported even though not used

Reply:
Might be better to put the import inside the block where it is used. The security issue is with the methods, not with the import.

Reply:
fair enough.